deployments/charts/kuma/values.yaml

```diff
@@ -93,6 +130,9 @@ controlPlane:
 
   # -- URL of Global Kuma CP
   globalZoneSyncService:
+    # -- Whether or not to create a k8s service for the global zone sync
+    # service. If null, it will be created only if mode is set to global.
+    enabled: null
```

Comment:
Why not have enabled default to be `true` and just ignore this option if it's a zone cp?

Reply:
Yeah that seems fine as well! I changed it to this.